docker-compose-https.yml

```diff
@@ -0,0 +1,76 @@
+version: "3.7"
```

Comment:
This file should be in the https folder, so `https/docker-compose.yml` (so we should see a diff here!)